.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,16 +6,16 @@
 
 ## Pre-launch Checklist
 
-- [ ] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
 - [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
 - [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
 - [ ] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
+- [ ] I signed the [CLA].
```

Comment:
New contributors generally do this after creating the PR since that's when the bot tells them to. I'm not sure whether the ordering is intended to be new or established contributors; I would have thought the former since for established contributors the checklist is much more rote.

Reply:
Well to be frank I put the order in the order that I usually complete them, but in any case, wouldn't the order be the same here for new or established contributors? New ones should hopefully read the contributor guide first, then the other two, before they write their PR, and certainly before they get it reviewed, but they don't need to sign the CLA until it's ready to land, really. And established contributors have already done all four.